spec/support/test_setup.rb

```diff
@@ -0,0 +1,94 @@
+# rubocop:disable Style/ClassVars
+class TestSetup
```

Comment:
iirc, you can just do this:

```
class TestSetup # rubocop:disable Style/ClassVars
```

and it'll only disable those checks for this class.